client/transport/Cargo.toml

```diff
@@ -1,13 +1,17 @@
 [package]
 name = "jsonrpsee-client-transport"
-version = "0.16.2"
-authors = ["Parity Technologies <admin@parity.io>", "Pierre Krieger <pierre.krieger1708@gmail.com>"]
 description = "JSON-RPC client transport"
-edition = "2021"
-license = "MIT"
-repository = "https://github.com/paritytech/jsonrpsee"
-homepage = "https://github.com/paritytech/jsonrpsee"
-documentation = "https://docs.rs/jsonrpsee-client-transport"
+version = "0.16.2"
```

Comment:
I'm not sure why version wasn't inherited in Subxt; if we want to keep it the same it seems to make sense to just inherit it too